docs/content/reference.md

```diff
@@ -151,6 +151,28 @@ Bolt's client is an instance of `WebClient` from the [Node Slack SDK](https://to
 
 :::
 
+## Assistants
+
+### The `AssistantConfig` configuration object
+
+| Property | Required? | Description | 
+|---|---|---|
+|`threadContextStore` | Optional, but recommended | When provided, must have the required methods to get and save thread context, which will override the `getThreadContext` and `saveThreadContext` utilities. <br/> <br/> If not provided, a `DefaultAssistantContextStore` instance is used.
```

Comment:
>but recommended

Actually, either is fine! The default one works well. If there is no concern, we can remove ", but recommended" from this part